src/control/cmd/daos_admin/handler.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/pbin"
+	"github.com/daos-stack/daos/src/control/server/storage"
```

Comment:
dunno much about go imports, but isn't that a superset of the 2 imports below that should be removed?

Reply:
All are actually needed - the file uses both scm and bdev storage providers from packages below.
Btw. I'm not an expert in GO either, but noticed the GOlang plugin in vscode does automatically take care of adding/cleaning up pkg list, so it should be all fine.

Reply:
so we have a custom preference which I don't think is handled by vscode (last time I checked), we prefer groupings by stdlib/vendor/local i.e. fmt in first group, protobuf in second group then daos/common in the third group, let me know if that doesn't make sense

Reply:
@johannlombardi it's not actually a superset, the child packages have their own namespaces (storage package is distinct from bdev and scm), it is a bit counterintuitive

Reply:
Is there a tool for that?

Reply:
No, imports are done on a package level. Each directory is its own package rather than the root of a hierarchy.